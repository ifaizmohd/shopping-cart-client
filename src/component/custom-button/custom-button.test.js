import { shallow } from "enzyme";
import React from "react";
import CustomButton from "./custom-button.component";

describe("Custom button component testing", () => {
  it("Shallow snapshot testing of custom button component", () => {
    expect(shallow(<CustomButton />)).toMatchSnapshot();
  });
});
