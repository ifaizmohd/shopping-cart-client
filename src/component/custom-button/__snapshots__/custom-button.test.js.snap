// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Custom button component testing Shallow snapshot testing of custom button component 1`] = `ShallowWrapper {}`;
